setup.cfg

```diff
@@ -1,18 +0,0 @@
-[flake8]
-exclude =
-    .git/,
-    __pycache__,
-    build/
-max-line-length = 100
-
-[tool:pytest]
-testpaths = tests
-addopts = -rsxX --showlocals --tb=native -p no:cacheprovider
-
-[tool:release]
-github_user = "mozilla-services"
-github_project = "socorro-submitter"
-bugzilla_product = "Socorro"
-bugzilla_component = "Infra"
-main_branch = "main"
-tag_name_template = "%%Y.%%m.%%d"
```

Comment:
`tool:pytest` and `tool:release` were moved to `pyproject.toml`.

`flake8` was removed and replaced with ruff configuration in `pyproject.toml`.